release/nightly_tests/dataset/app_config.yaml

```diff
@@ -1,12 +1,12 @@
-base_image: "anyscale/ray-ml:pinned-nightly-py37-gpu"
+base_image: "anyscale/ray-ml:nightly-py37-gpu"
 
 python:
-  pip_packages: []
+  pip_packages:
+    - boto3
+    - pyarrow
+    - tqdm
   conda_packages: []
 
 post_build_cmds:
-  - pip uninstall -y numpy ray || true
```

Comment:
Maybe keep ray uninstall?

Reply:
Is it necessary? Tests look to be passing fine without it.

Reply:
I think when you provide the custom wheel url, this is needed iirc, -U doesn’t guarantee to redownload the wheel if the version is the same (which is the case for the custom url, both are mostly version 2.0)

Reply:
Ah I see- I added it back to all of these.